src/current/_data/redirects.yml

```diff
@@ -819,6 +809,25 @@
     - interleave-in-parent.md
     - stable/interleave-in-parent.md
 
+# Cloud 2.0 updates (Serverless/Dedicated to Basic/Standard/Advanced)
```

Comment:
Nit: I'd remove this comment before merging.

Reply:
I meant it as a header to help us add and track redirects more easily for Cloud 2.0